spec/system/claims/view_school_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+RSpec.feature "School Page" do
+  after { Capybara.app_host = nil }
+
+  scenario "User visits his school" do
+    persona = given_there_is_an_existing_persona_for("Mary")
+    given_i_am_on_the_claims_site
+    when_i_visit_home_page
+    when_i_click_sign_in(persona)
+    when_i_visit_a_school_page
+    i_can_see_organisation_details
+    i_can_see_contact_details
+  end
+
+  private
+
+  def given_i_am_on_the_claims_site
+    Capybara.app_host = "http://#{ENV["CLAIMS_HOST"]}"
+  end
```

Comment:
This is already handled by the services test helper that is included into System specs.